src/main/kotlin/org/jitsi/utils/JavaVersion.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:JvmName("JavaVersion")
```

Comment:
Being paranoid: did you test that this works?  I thought I remembered it insists on being the very first line of the file (even above comments), but I could be wrong.

Reply:
I tested in the IDE, it had to be before the `package` line.